neo/Network/P2P/Payloads/Transaction.cs

```diff
@@ -175,8 +175,8 @@ public JObject ToJson()
             json["version"] = Version;
             json["nonce"] = Nonce;
             json["sender"] = Sender.ToAddress();
-            json["sys_fee"] = new BigDecimal(SystemFee, NativeContract.GAS.Decimals).ToString();
-            json["net_fee"] = new BigDecimal(NetworkFee, NativeContract.GAS.Decimals).ToString();
+            json["sys_fee"] = SystemFee.ToString();
+            json["net_fee"] = NetworkFee.ToString();
             json["valid_until_block"] = ValidUntilBlock;
```

Comment:
in `FromJson` are good parsed, so obviously here is wrong

Reply:
`FromJson()` was added with the RPC SDK. So it is inconsistent. But it's fine now.